docs/search/get-started-with-search/search-basics/reference-field-special-characters.md

```diff
@@ -25,7 +25,7 @@ You can create a field with the `as` operator.
 
 You don't need to escape field names that:
 
-* have characters matching `a-zA-Z0-9_`
-* don't begin with a number
+* have characters matching `a-zA-Z0-9_.`
+* don't begin with a number or `.(dot)`
```

Comment:
```suggestion
* don't begin with a number or dot (`.`)
```

Reply:
> 

```suggestion
* don't begin with a number or `. ` (dot)
```

Reply:
- Here, in the first line, we have added " . " at the end of this limitation. 
- In the Second line " . " is the char we have added.  
 (dot) is just a way to explain the " . " char, so this should come in parenthesis, which we use to describe additional statements or words.